jnim/private/jni_export.nim

```diff
@@ -399,8 +399,9 @@ macro jexport*(a: varargs[untyped]): untyped =
 
         thunkCall = newCall(bindSym"nimValueToJni", envName, thunkCall, copyNimTree(thunkParams[0]))
 
-        let thunk = newProc(thunkName, thunkParams)
+        let thunk = newProc(postfix(thunkName, "*"), thunkParams)
```

Comment:
This should not be necessary. `dynlib` and `exportc` should be enough to keep the symbol around.

Reply:
Hm, I will double-check this, may be that is indeed a leftover from the experimentation phase?